Fortinet/fortigate/ingest/parser.yml

```
@@ -241,6 +241,7 @@ stages:
           fortinet.fortigate.attack.id: "{{parsed_event.message.attackid}}"
           fortinet.fortigate.policyid: "{{parsed_event.message.policyid}}"
           fortinet.fortigate.poluuid: "{{parsed_event.message.poluuid}}"
+          fortinet.fortigate.forwardedforip: "{{parsed_event.message.forwardedfor}}"
```

[network.forwarded_ip](https://www.elastic.co/guide/en/ecs/8.11/ecs-network.html#field-network-forwarded-ip) is more suitable:

```suggestion
          network.forwarded_ip: "{{parsed_event.message.forwardedfor}}"
```